app/services/stripe_model_callbacks/invoice/updated_service.rb

```diff
@@ -1,18 +1,18 @@
 class StripeModelCallbacks::Invoice::UpdatedService < StripeModelCallbacks::BaseEventService
   def execute
     invoice.assign_from_stripe(object)
+    return success_actions if invoice.save
```

Comment:
Irrelevant changes I would argue actually increased the abstractness of the code without solving a complexity offence for example :-)

Reply:
It kinda good practice from ruby style guide - https://github.com/rubocop-hq/ruby-style-guide#nested-conditionals.

Reply:
For complex operations I would agree, but this isn't the case here. Its also not relevant to this PR.